### DISCUSSION TITLE
Variation compare, differences and which is better?

### DISCUSSION THREAD

**Discussion by AndrewTXJ** [egzumer/uv-k5-firmware-custom#360] [Polls]
Did anyone simultaneously have three of those variation?
UV-K5
UV-K6 / UV-K5(8)
UV-5R Plus

What's actually the difference of this three beside the very obvious one like the backlight and the shape design, i have using K6 for couple month now a month on stock K6 firmware, a month for others variation stock firmware from quansheng, and couple moth for so many different mod, and finally end up in this custom firmware, and even try to modify some easier code like turning things on and off, and change some others like roger beep tone and the startup version string, my friends considering buying one, but don't know what's the different, i am considering buying another too, so i can have one for stable everyday use, one for testing all kind of modification, does this three really have different? did the K6's speaker is really better then K5 like the quansheng advertise? if there is really some different, which one will you chose or which one do you think is better?

BTW, i know there is others brand that take this radio from quansheng and slap their logo on it, but i assume everyone with the same design is the same as the original quansheng version, eg. the retevis RA79 is basically a K5,and others, but if anyone have variation from others brand and experience that there has different, please let me know, so i can take it to my consideration.

**Comment by egzumer** [comment]
Those 3 are the same radio. The only real difference is that 5R have no USB charging, which for me is a no buy.
There are differences in mainboard versions. Old one was 1.4, the newest is 1.6.
I had 
- 1x K5, first one I had. It was 1.4. Don't know what version are sold now.
- 4x 5R, all were 1.4, last one was some time ago so I don't know what are the current versions
- ~8x K6, at the beginning I got 1.4 then I got one that was 1.4 the other 1.6, the last ones were all 1.6

I don't know if that is model dependent, maybe they are all 1.6 now.

The audio is different between boards, not radio models. There were differences even between 1.4 boards, different resistors were used in audio path. The speaker itself is the same. The advertisement is a lie.
1.6 has different charging chip. Also if you are going to use speaker-mic you have to check it for a [short](https://github.com/egzumer/uv-k5-firmware-custom/discussions/267) that is present in most of the radios with this board.

**Reply by AndrewTXJ** [reply]
So 1.6 is the one that has better audio, how do i know which version is it?

**Reply by egzumer** [reply]
Some 1.4 has also better radio.
https://github.com/ludwich66/Quansheng_UV-K5_Wiki/wiki/PCB-V1.6

**Reply by AndrewTXJ** [reply]
1.4 has better radio? not the 1.6?

**Reply by egzumer** [reply]
Some 1.4, and I guess all 1.6

**Reply by AndrewTXJ** [reply]
so, how to determine the version of the radio?

**Reply by egzumer** [reply]
Better audio, not radio. Who said it has better radio, it is the same board, why would it have better radio?

Edit: Eeeh, I said it, I meant audio, the sound is lower pitch:)

**Comment by Hunterfnet** [comment]
I dunno why maybe chips control quality are better on production , but i can assure the UV5R+ headphone sounds much better and receives better than UVK6 -5(8) . On other way the USB charger added on uvk6 dont serve for charge 100% the battery anyone can make the test first .
I can't say nothing about uvk5 i don't liked the dessign and dont have it for compare.

**Reply by egzumer** [reply]
1.6 charges to 100%. Again there is no difference between K5/K6/5R, there are differences between boards which is in no way related to a housing.

**Comment by ErikS-web** [comment]
[fcc.report.FCC-ID.XBPUV-K5.6401581.pdf](https://github.com/egzumer/uv-k5-firmware-custom/files/13326497/fcc.report.FCC-ID.XBPUV-K5.6401581.pdf)
 
> _Subject: Explanation of differences between product types
> FCC ID : XBPUV-K5
> Model Number: **UV-K5, UV-K5(2), UV-K5(6), UV-K5(8), UV-K5(9), UV-K5(11),
> UV-K5(22), UV-K5(66), UV-K5(88), UV-K5(99), UV-5R, UV-5R PLUS, UV-82**
> To Whom It May Concern:
> We, The QUANSHENG ELECTRONICS CO., LTD. hereby declare that the models are
> identical except the model names & cover designs.
> Except for the different above, no other modification is performed._
> 
>

**Reply by AndrewTXJ** [reply]
Cleary not true, the uv5r plus doesn't have usb charging, so it;s not only model name and cover design.

**Comment by ErikS-web** [comment]
The Egzumer firmare here on this Github, can function on all types.
There is no difference in functionality. Only appearance, such as color of display and feeling.

If your motivation is that you want to understand it electronically and you want to know everything as precisely as possible, then read up carefully. Everything has already been written on internet sites and Telegram and here too there are references to those sites where all hardware is described in detail. With even component lists, component specifications, X-ray photos, parts diagrams, etc.

**Comment by okcrum** [comment]
I have the UV-K6 and uv-K5(99) <--- not a typo but rather 1.6 rev. board (newer USB charger chip) in a new case with what look like somewhat better seals. Oh, and they come in 4 trim colors. :)

The K6s are 1.4 rev. boards.

@andrewtanyinkit, the firmware should tell you the hardware rev., either at powerup or in one of the menu items.
